### DISCUSSION TITLE
Live example apps using glide data grid

### DISCUSSION THREAD

**Discussion by nkeat12** [glideapps/glide-data-grid#826] [General]
Looking for live apps that use glide that I can play with, anyone have a list or know of any? The storybook examples are great but its always nice to see real implementations from real businesses/apps.

**Comment by lukasmasuch** [comment]
The data grid is implemented as part of Streamlit, an open-source library to build data apps in Python. For example: https://docs.streamlit.io/library/api-reference/data/st.data_editor#configuring-columns
